src/KeycloakGuardServiceProvider.php

```diff
@@ -18,7 +18,11 @@ public function boot()
   public function register()
   {
     Auth::extend('keycloak', function ($app, $name, array $config) {
-      return new KeycloakGuard(Auth::createUserProvider($config['provider']), $app->request);
+      return new KeycloakGuard(
+        Auth::createUserProvider($config['provider']),
+        $app->request,
+        $config['input_key'] ?? 'api_token'
```

Comment:
Maybe you forgot to add this 'input_key' on default config file?

Reply:
Ah I totally missed the correct config, I refactored to use the correct config!